bigquery/test/tables.test.js

```diff
@@ -418,148 +394,101 @@ describe('bigquery:tables', function () {
-
-      program.main(['import', dataset, table, file]);
-      assert.equal(program.importFile.calledOnce, true);
-      assert.deepEqual(program.importFile.firstCall.args.slice(0, -1), [{
-        dataset: dataset,
-        table: table,
-        file: file,
-        bucket: undefined,
-        format: undefined
-      }]);
+      program.importLocalFile = sinon.stub();
+
+      program.main(['import', datasetId, tableId, fileName]);
+      assert.equal(program.importLocalFile.calledOnce, true);
+      assert.deepEqual(program.importLocalFile.firstCall.args.slice(0, -1), [datasetId, tableId, fileName]);
+    });
+
+    it('should call importFileFromGCS', function () {
+      var program = getSample().program;
+      program.importFileFromGCS = sinon.stub();
+
+      program.main(['import', datasetId, tableId, fileName, '-b', bucketName]);
+      assert.equal(program.importFileFromGCS.calledOnce, true);
+      assert.deepEqual(program.importFileFromGCS.firstCall.args.slice(0, -1), [datasetId, tableId, bucketName, fileName]);
     });
 
     it('should call copyTable', function () {
       var program = getSample().program;
       program.copyTable = sinon.stub();
 
-      program.main(['copy', srcDataset, srcTable, destDataset, destTable]);
+      program.main(['copy', srcDatasetId, srcTableId, destDatasetId, destTableId]);
       assert.equal(program.copyTable.calledOnce, true);
-      assert.deepEqual(program.copyTable.firstCall.args.slice(0, -1),
-        [srcDataset, srcTable, destDataset, destTable]
-      );
+      assert.deepEqual(program.copyTable.firstCall.args.slice(0, -1), [srcDatasetId, srcTableId, destDatasetId, destTableId]);
     });
 
     it('should call exportTableToGCS', function () {
       var program = getSample().program;
       program.exportTableToGCS = sinon.stub();
 
-      program.main(['export', dataset, table, bucket, file]);
+      program.main(['export', datasetId, tableId, bucketName, fileName]);
       assert.equal(program.exportTableToGCS.calledOnce, true);
-      assert.deepEqual(program.exportTableToGCS.firstCall.args.slice(0, -1), [{
-        dataset: dataset,
-        table: table,
-        file: file,
-        bucket: bucket,
-        format: undefined,
-        gzip: false
-      }]);
+      assert.deepEqual(program.exportTableToGCS.firstCall.args.slice(0, -1), [datasetId, tableId, bucketName, fileName]);
     });
 
     it('should call insertRowsAsStream', function () {
       var program = getSample().program;
       program.insertRowsAsStream = sinon.stub();
 
-      program.main(['insert', dataset, table, validJsonFile]);
+      program.main(['insert', datasetId, tableId, validJsonFile]);
 
       assert.equal(program.insertRowsAsStream.calledOnce, true);
-      assert.deepEqual(
-        program.insertRowsAsStream.firstCall.args.slice(0, -1),
-        [{ rows: jsonArray, dataset: dataset, table: table }]
-      );
-    });
-
-    it('should recognize --gzip flag', function () {
```

Comment:
I deleted this test because I deleted the gzip option from the sample to simplify the sample. We kinda started going overboard on our CLI programs, but easy-to-understand samples for the docs take priority, so I removed gzip from the sample.

Perhaps we can advocate for a new sample (that the other languages would also need to implement) that specifically shows how to do gzipping.